crates/oci-distribution/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "oci-distribution"
+version = "0.1.0"
+authors = [
+    "Matt Butcher <matt.butcher@microsoft.com>",
+    "Matthew Fisher <matt.fisher@microsoft.com>",
+    "Radu Matei <radu.matei@microsoft.com>",
+    "Taylor Thomas <taylor.thomas@microsoft.com>",
+    "Brian Ketelsen <Brian.Ketelsen@microsoft.com>",
+    "Brian Hardock <Brian.Hardock@microsoft.com>",
+    "Ryan Levick <rylevick@microsoft.com>",
+]
+edition = "2018"
+
+[dependencies]
+reqwest = { version = "0.10", features = ["json"] }
+anyhow = "1.0.25"
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
We shouldn't include full features if possible. This significantly hurts compile times.

Reply:
I'm confused on this one... the async_oauth2 said it needed the above. Do I just ignore the docs in that case? (At this point, I removed async_oauth, so I think I can go back to whatever)